cmake/kokkos_tribits.cmake

```diff
@@ -244,6 +244,15 @@ FUNCTION(KOKKOS_SET_LIBRARY_PROPERTIES LIBRARY_NAME)
     ${LIBRARY_NAME} PUBLIC ${KOKKOS_LINK_LIBRARIES}
   )
 
+  IF (KOKKOS_ENABLE_OPENMP)
+
+    TARGET_LINK_LIBRARIES(
```

Comment:
If we do this, we should call `find_package(OpenMP)` before always. Otherwise, the target is not defined.

Reply:
I generally think we should move to using the FindOpenMP.